### PR TITLE
Update ros examples

### PR DESCRIPTION
**Pull request by Levi-Armstrong**
- [x] Basic Cartesian Example
- [x] Car Seat Example
- [x] Freespace Hybrid Example
- [x] Freespace OMPL Example
- [x] Glass Upright Example
- [x] Glass Upright OMPL Example (Will not fix)
- [x] Online Planning Example
- [x] Pick and Place Example
- [x] Puzzle Piece Auxiliary Axes Example
- [x] Puzzle Piece Example
- [x] Scene Graph Example

### CODE REVIEW

**Comment by Levi-Armstrong** [comment]
This includes the last two examples remaining to be updated to the command language. It depend on Tesseract [PR](https://github.com/ros-industrial-consortium/tesseract/pull/497)

**Comment by Levi-Armstrong** [comment]
@mpowelson I update the online planner because of changes in trajopt but not sure if it was done correctly. Also is `collision_config.type = CollisionEvaluatorType::DISCRETE;` required in the config if we have evaluators for each?

**Comment by Levi-Armstrong** [comment]
I was not sure where to set the safety_margin_buffer.

**Comment by Levi-Armstrong** [comment]
@mpowelson  I believe this can be merged, just have the one question above before I do.

File: tesseract_ros/tesseract_examples/tesseract_ros_examples/src/online_planning_example.cpp [hunk 1]
```diff
@@ -210,21 +210,13 @@ bool OnlinePlanningExample::setupProblem()
...
-        margin_data,
-        tesseract_collision::ContactTestType::CLOSEST,
-        var_vector,
-        trajopt::CollisionExpressionEvaluatorType::SINGLE_TIME_STEP,
-        safety_margin_buffer,
-        true);
+    trajopt::TrajOptCollisionConfig collision_config(margin, margin_coeff);
+    collision_config.contact_request.type = tesseract_collision::ContactTestType::CLOSEST;
+    collision_config.type = CollisionEvaluatorType::DISCRETE;
+    double safety_margin_buffer = 0.10;  // Where does this go with new interface?
```
Comment (mpowelson) [inline tesseract_ros/tesseract_examples/tesseract_ros_examples/src/online_planning_example.cpp:216]: @Levi-Armstrong It should be collision_config.collision_margin_buffer

**Review by mpowelson — APPROVED** [review]

**Comment by mpowelson** [comment]
Just that one thing that needs to be updated.

**Comment by Levi-Armstrong** [comment]
> > @mpowelson I update the online planner because of changes in trajopt but not sure if it was done correctly. Also is `collision_config.type = CollisionEvaluatorType::DISCRETE;` required in the config if we have evaluators for each?
> 
> Somehow missed all of these. trajopt_ifopt needs to be updated to support the other evaluator types. Eventually I assume it will take the base class and use this to switch between them.

Thank you.

**Pull request merged**